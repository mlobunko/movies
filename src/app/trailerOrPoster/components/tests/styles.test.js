import React from 'react';
import { shallow } from 'enzyme';
import 'enzyme-to-json';

import {
  StyledPoster,
  StyledPosterContainer,
  StyledPosterContent,
  StyledTrailer,
  StyledTrailerContainer,
  StyledTrailerContent
} from '../styles';

const posterPath = 'abc';
const trailerPath = 'cba';

describe('StyledPoster', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledPoster posterPath={posterPath} />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('StyledPosterContainer', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledPosterContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('StyledPosterContent', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledPosterContent />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('StyledTrailer', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledTrailer trailerPath={trailerPath} />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('StyledTrailerContainer', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledTrailerContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('StyledTrailerContent', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledTrailerContent />);
    expect(wrapper).toMatchSnapshot();
  });
});
