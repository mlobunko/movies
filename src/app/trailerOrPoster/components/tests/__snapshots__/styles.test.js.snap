// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledPoster renders correctly 1`] = `
<div
  className="sc-bdVaJa dLEHil"
/>
`;

exports[`StyledPosterContainer renders correctly 1`] = `
<div
  className="sc-htpNat byZALs"
/>
`;

exports[`StyledPosterContent renders correctly 1`] = `
<div
  className="sc-ifAKCX bGkqyn"
/>
`;

exports[`StyledTrailer renders correctly 1`] = `
<iframe
  allow="autoplay; encrypted-media"
  allowFullScreen={true}
  className="sc-bwzfXH isPazB"
  frameBorder="0"
  src="https://www.youtube.com/embed/cba?rel=0&amp;showinfo=0"
/>
`;

exports[`StyledTrailerContainer renders correctly 1`] = `
<div
  className="sc-bxivhb ivXFnf"
/>
`;

exports[`StyledTrailerContent renders correctly 1`] = `
<div
  className="sc-EHOje fjEyDX"
/>
`;
